src/main/java/com/db/am/bauhaus/project/EtsyApiActions.java

```diff
@@ -0,0 +1,57 @@
+package com.db.am.bauhaus.project;
+
+import static net.serenitybdd.rest.SerenityRest.rest;
+import static net.serenitybdd.rest.SerenityRest.then;
+import static net.serenitybdd.rest.SerenityRest.when;
+import static net.serenitybdd.rest.SerenityRest.and;
+import static net.serenitybdd.rest.SerenityRest.given;
+import net.thucydides.core.annotations.Step;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class EtsyApiActions {
+
```

Comment:
I realise now I understand Serenity more, that this is a Step Library really. It should therefore have been created in the steplib directory with, for example, SearchUser.